src/python/turicreate/data_structures/sframe.py

```diff
@@ -2189,6 +2189,12 @@ def _imagecols_to_stringcols(self, num_rows=10):
                     printed_sf[t] = self[t].astype(str)
         return printed_sf.head(num_rows)
 
+    def drop_duplicates(self, subset=None, keep='first', inplace=False): 
+        self=pandas.DataFrame(self) 
+        self=self.drop_duplicates(subset=subset, keep=keep, inplace=inplace) 
+        self=load_sframe(self) 
+        return self
+
```

Comment:
This would work, but we should have this function wrap the  groupby SELECT_ONE operator to perform this.